xbmc/epg/EpgDatabase.cpp

```diff
@@ -99,6 +105,17 @@ void CEpgDatabase::UpdateTables(int iVersion)
 
   if (iVersion < 9)
     m_pDS->exec("ALTER TABLE epgtags ADD sIconPath varchar(255);");
+  
+  if (iVersion < 10)
```

Comment:
Hehe well IMO SQLite only has limited SQL support when it comes to table management but it supports a lot more stuff when it comes to sub-selects, exists etc.
As most people will be using an SQLite database I always develop using SQLite. It's also much easier to backup/drop during development than MySQL.